packages/analytics/src/api.ts

```diff
@@ -224,6 +225,23 @@ export function setAnalyticsCollectionEnabled(
     enabled
   ).catch(e => logger.error(e));
 }
+
+/**
+ * Adds data that will be set on every event logged from the SDK, including automatic one.
```

Comment:
@egilmorez Can you review the text here? This comment here should be the only text that will end up in the reference docs.

From me:
- one => ones
- In "gtag's set command" the "set" should be in quotes or backticks, I think quotes because it's a string not a var